tools/project.py

```diff
@@ -128,77 +125,68 @@
         if exists(EXPORT_DIR):
             rmtree(EXPORT_DIR)
 
+
+    # Target
+    if options.mcu is None :
+        args_error(parser, "[ERROR] You should specify an MCU")
+    mcus = options.mcu
+
+    # IDE
+    if options.ide is None:
+        args_error(parser, "[ERROR] You should specify an IDE")
+    ide = options.ide
+
+    # Program Number or name
+    p, n, src= options.program, options.program_name, options.source_dir
```

Comment:
I tried to keep as much of the original project.py; this was already the naming scheme.